test/unit/messages.js

```diff
@@ -144,6 +144,25 @@ exports['addMessage supports template variables on doc'] = function(test) {
     test.done();
 };
 
+exports['addMessage does not escape characters - #3795'] = function(test) {
+    var doc = {
+        form: 'x',
+        reported_date: '2050-03-13T13:06:22.002Z',
+        place: 'Sharon\'s Place'
```

Comment:
I'm surprised `'` actually gets escaped at all, but checking mustache source it does:
```
  var entityMap = {
    '&': '&amp;',
    '<': '&lt;',
    '>': '&gt;',
    '"': '&quot;',
    "'": '&#39;',
    '/': '&#x2F;',
    '`': '&#x60;',
    '=': '&#x3D;'
  };
```
Might be worth adding a couple of `>`, `<` or `&` to the example to make it more obvious.